Shared/DateGroupedTableData.swift

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is this sourced from somewhere?

Reply:
It's copy pasted from https://github.com/mozilla-mobile/firefox-ios/blob/master/Shared/DateGroupedTableData.swift